src/_sass/_site.scss

```diff
@@ -25,6 +25,19 @@
 $font-color: bootstrap.$body-color;
 $font-weight-light-bold: (bootstrap.$font-weight-normal + bootstrap.$font-weight-bold) * 0.5;
 
+// Reset initial values
+h1, h2, h3, h4, h5, h6, p, ol, ul {
+  display: block;
+  margin-block-start: 0px;
+  // margin-block-end: 0px;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+  padding-block-start: 0px;
+  padding-block-end: 0px;
+  // padding-inline-start: 0px;
+  padding-inline-end: 0px;
+  unicode-bidi: isolate;
+}
```

Comment:
Why does this change require changing the margin of all of the headers on the site?

Reply:
All browsers have default settings around margins, etc. If you don't reset them, your styles don't necessarily do what you expect.

Reply:
I understand how default CSS styles work, I'm asking why do you need to reset _all_ of the styles for the entire site, when our CSS is assuming the browser defaults? This seems like it's impacting other areas of the site that you didn't intend to change.